src/dynamo.js

```diff
@@ -37,7 +37,7 @@ export class Dynamo {
             })
             );
         let params = { RequestItems: { OnlineUsers: userDeleteRequests } };
-        await client.batchWriteItem(params).catch(e => console.log(e));
+        await client.batchWriteItem(params).catch(() => console.log('All expired users are deleted'));
```

Comment:
Is this true? On catch it might be that not all are deleted?

Reply:
Yep, this is broken. I had a call with @iva-stoycheva how this can be fixed